circleguard/comparer.py

```diff
@@ -144,7 +149,33 @@ def _compare_two_replays(replay1, replay2):
         if (Mod.HR in replay1.mods) ^ (Mod.HR in replay2.mods):
             xy1[:, 1] = 384 - xy1[:, 1]
 
-        return Comparer._compute_data_similarity(xy1, xy2)
+        (play_sequence_1, play_sequence_2) = Comparer.convert_to_row_major_form(xy1, xy2)
+
+        # Sectioned into 20 chunks, used to ignore outliers (eg. long replay with breaks is copied, and the cheater cursordances during the break)
```

Comment:
20 was arbitrarily chosen, whether to section maps into constant chunks or a constant number of chunks is also up for debate.

Reply:
Even for tiny maps, I think we have enough data points such that slicing it into chunks of size 20 is likely still okay.

Sample rate is generally about 40-60hz.